install_files/ansible-base/roles/tails-config/files/securedrop_init.py

```diff
@@ -48,7 +50,27 @@
                  '/org/gnome/nautilus/preferences/automatic-decompression',
                  'false'])
 
-# notify the user
+# Set journalist.desktop and source.desktop links as trusted with Nautilus (see
+# https://github.com/freedomofpress/securedrop/issues/2586)
+# set euid and env variables to amnesia user
+os.setresgid(1000, 1000, -1)
```

Comment:
In order to avoid magic numbers, can we do something like: 

```
import pwd, grp

amnesia_gid = grp.getgrnam("amnesia").gr_gid
os.setresgid(amnesia_gid, amnesia_gid, -1)
amensia_uid = pwd.getpwnam("amnesia").pw_uid
os.setresuid(amensia_uid, amensia_uid, -1)
```